docker-compose.yml

```diff
@@ -1,6 +1,29 @@
 version: '3.8'
 
 services:
+  postgres:
+    image: postgres:9.6
+    restart: unless-stopped
+    environment:
+      - POSTGRES_USER=keystone
+      - POSTGRES_PASSWORD=keystone
+      - POSTGRES_DB=keystone
+    ports:
+      - '5432:5432'
+
+  keystone:
+    image: keystone:latest
```

Comment:
should we fix a version number here?

Reply:
`latest` refers to the latest docker image that we build in the `docker-compose` process, so we do want to use the `latest` in this case.